src/application-intl/index.js

```diff
@@ -0,0 +1,4 @@
+import ApplicationIntlContext from './ApplicationIntlContext';
+import ApplicationIntlProvider from './ApplicationIntlProvider';
+
+export { ApplicationIntlContext, ApplicationIntlProvider };
```

Comment:
Should we be exporting the context shape as well?

Reply:
Yes, good call